documentation/source/install/on_premise_solutions.rst

```diff
@@ -96,6 +96,301 @@ The *ubuntu@kubemaster-01:~$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.
 * Create an OpenEBS Volume and mount to the Jupyter Server Pod (/mnt/data) (kubectl get pvc) (kubectl get pv) (kubectl get pods)
 * Expose the Jupyter Server to external world through http://NodeIP:8888 (NodeIP is any of the nodes' external IP) (kubectl get pods)
 
+Installing Kubernetes on CentOS in Vagrant VMs
+----------------------------------------------------
+The following procedure helps you install Kubernetes on CentOS version 7.4 and use OpenEBS on a local machine. You will be setting up a 3 node cluster comprising of 1 Master and 2 Worker Nodes running Kubernetes 1.8.5.
```

Comment:
OpenEBS on a local machine = OpenEBS on that cluster.

Reply:
Done!